www/src/components/Settings.tsx

```diff
@@ -85,9 +90,31 @@ const Settings = () => {
             ))}
           </Form.Control>
         </Form.Group>
+        {!showMinimizedTitle && (
+          <Form.Group>
+            <Form.Control
+              as="select"
+              value={settings.containerWidth}
+              onChange={(e: React.ChangeEvent<HTMLSelectElement>) => handleSettingsChange('containerWidth', e.target.value)}
+              floatingLabel="Container width"
```

Comment:
nit: "Container **W**idth" to match casing of other labels.

Reply:
Corrected